modules/virt-resources-monitored-actively-in-Virtualization-Overview-page.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assembly:
+//
+// * virt/logging_events_monitoring/virt-resources-monitored-actively-in-Virtualization-Overview-page.adoc
+//
+
+[id="virt-resources-monitored-actively-in-Virtualization-Overview-page_{context}"]
+= Resources monitored actively in the Virtualization Overview page
+
+The following table shows actively monitored resources in the *Virtualization Overview* page. This information is useful when you need obtain relevant data and intervene to resolve a problem.
```

Comment:
Suggestion: "The following table shows metrics displayed on the **Virtualization Overview** page. You can use these metrics to obtain relevant data and to resolve any issues."

Reply:
Revamped as follows:
"The following table shows actively monitored resources, metrics, and fields in the *Virtualization Overview* page. This information is useful when you need obtain relevant data and intervene to resolve a problem."

Permissions does not fall under metrics.